src/transport_performance/urban_centres/raster_uc.py

```diff
@@ -103,29 +103,29 @@ def get_urban_centre(
         centre : tuple
             Tuple with coordinates for city centre. Urban centres that do not
             contain these coordinates will be filtered out.
-        centre_crs : str = None, optional
+        centre_crs : str, optional
             crs string of the centre coordinates. If None, it will default to
             raster_crs.
-        band_n : int = 1, optional
+        band_n : int, optional
             Band number to load from the geoTIFF.
-        cell_pop_threshold : int = 1500, optional
+        cell_pop_threshold : int, optional
             When calculating urban centre, keep cells whose value is equal or
             higher than the threshold.
-        diag : bool = False, optional
+        diag : bool, optional
             When calculating clusters, if True, diagonals are considered as
             adjacent and included in the cluster.
-        cluster_pop_threshold : int = 50000, optional
+        cluster_pop_threshold : int, optional
             Threshold to consider inclusion of cluster. If total population in
             cluster is lower than threshold, the cluster label is filtered out.
-        cell_fill_threshold : int = 5, optional
+        cell_fill_threshold : int, optional
             Defines gap filling behaviour. If the number of cells adjacent to
             any empty cell belonging to a cluster is higher than the threshold,
             the cell is filled with the cluster value. Needs to be between 5
             and 8.
-        vector_nodata : int = -200, optional
+        vector_nodata : int, optional
             Value to fill empty cells. Select a negative value that you would
             not expect to encounter within the raster population data.
-        buffer_size : int = 10000, optional
+        buffer_size : int, optional
```

Comment:
@SergioRec if we're removing the default value, then we need to remove the optional flag too, right?